src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
 ```
 [This article](/how-to-distribute-signed-http-exchanges/) illustrates how to use nginx to distribute SXGs.
 
+## Unique Advantages
+
+SXG is one of many possible technologies to enable [cross-origin prefetching](https://developer.chrome.com/blog/cross-origin-prefetch/). When deciding which technology to use, you may need to trade off between optimizing different aspects. The following sections illustrate a few of the unique values that SXG provides in the space of possible solutions.
+
+### Fewer requests to serve
+
+With cross-site prefetching, your server may need to serve additional requests. This corresponds to cases where a page was prefetched, but either the user didn't visit the page, or the prefetched bytes couldn't be shown to the user. For SXG, this additional traffic can be significantly reduced:
```

Comment:
Given that the audience is both websites and CDN partners,, what do you think about framing this along the lines of fewer unnavigated/ wasted requests to serve. Also calling out that over time we expect a better user experience from improved performance to mean more actual navigations to sites

Reply:
OK, renamed to "additional unused requests". I worry "wasted" may be too negative -- no prediction is 100% precision.